.docfx/packages/index.md

```
@@ -1,19 +1,11 @@
 # NuGet Packages
 
-This is a list of all NuGet packages from **ClassLibrary1** that is publicly available on [NuGet.org](https://www.nuget.org/packages?q=ClassLibrary1); the packages here are listed alphabetically and are available in preview-, rc- and production-ready versions.
+This is a list of all NuGet packages from **Unitify API by Codebelt** that is publicly available on [NuGet.org](https://www.nuget.org/packages?q=Codebelt.Unitify); the packages here are listed alphabetically and are available in preview-, rc- and production-ready versions.
 
-## 📦 Standalone Packages
+### 📦 Standalone Packages
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)